src/components/CheckBox/CheckBox.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import CheckBox from './CheckBox';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta = {
+  title: 'Component/CheckBox',
+  component: CheckBox,
+  tags: ['autodocs'],
+  parameters: {
+    layout: 'centered',
+    docs: {
+      description: {
+        component: 'CheckBox는 태그를 표시하는 컴포넌트입니다',
+      },
+    },
+  },
+  argTypes: {
+    className: {
+      control: { type: 'text' },
+      description: '추가적인 클래스를 지정합니다',
+      table: {
+        type: { summary: 'string' },
+      },
+    },
+    size: {
+      options: ['medium', 'large'],
+      control: { type: 'select' },
+      defaultValue: { summary: 'medium' },
+      description: '크기를 지정합니다',
+    },
+    disabled: {
+      options: [true, false],
+      control: { type: 'select' },
```

Comment:
boolean 값은 해당 코드로 작성해주시는 게 공식문서에 기반한 작성이 될 것 같습니다!
제가 놓쳤네요 죄송합니다ㅠㅠ

```suggestion
      control: boolean
```